cc/core/core.cc

```diff
@@ -158,3 +161,29 @@ NAN_METHOD(Core::PolarToCartAsync) {
     info
   );
 }
+
+NAN_METHOD(Core::GetNumThreads) {
+  FF_METHOD_CONTEXT("Core::GetNumThreads");
+  v8::Local<v8::Integer> ret = v8::Integer::New(v8::Isolate::GetCurrent(),
+      cv::getNumThreads());
+  FF_RETURN(ret);
```

Comment:
In general we try to use the converter classes where possible. This hopefully also helps, to reduce the refactoring that has to be done, in case we switch to NAPI one day:
``` c++
FF_RETURN(IntConverter::wrap(cv::getNumThreads()));
```